### PR TITLE
Configure pylint for Google's Python style.

### PR DESCRIPTION
**Pull request by jmichelp**
Adding pylint configuration, VSCode configuration to lint on save and
make the deploy.py script compliant.

Related to issue #50 

- [x] Tests pass (`deploy.py` is not tested yet)
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
...
     "--style=chromium"
   ],
+  "python.linting.enabled": true,
+  "python.linting.lintOnSave": true,
+  "python.linting.pylintEnabled": true,
   "[python]": {
     "editor.tabSize": 2
   },
-}
+}
```
Comment (ia0) [inline .vscode/settings.json]: Looks like a typo. It's usually a good idea to end files with a newline.

File: deploy.py [hunk 2]
```diff
@@ -19,12 +19,13 @@
...
 
 import argparse
-import colorama
 import copy
 import os
 import shutil
 import subprocess
 import sys
+
+import colorama
```
Comment (ia0) [inline deploy.py]: Out of curiosity (don't know Python), why do we need to reorder this import?

Comment (jmichelp) [reply deploy.py]: Just a style guide thing: system imports first. And colorama is a separate package so it belongs to the 2nd set of imports.

File: .vscode/settings.json (hunk 1 above)
Comment (jmichelp) [reply .vscode/settings.json]: The newline is being removed by the auto-formater in VSCode :/

File: deploy.py (hunk 2 above)
Comment (ia0) [reply deploy.py]: Oh I see. Makes sense. Thanks!

File: .vscode/settings.json (hunk 1 above)
Comment (ia0) [reply .vscode/settings.json]: I see. So it's fine since this file is meant to be edited by VSCode users anyway :-)

Comment (jmichelp) [reply .vscode/settings.json]: Found the culprit! Configuration mismatch between my workstation and my laptop :)

Comment (ia0) [reply .vscode/settings.json]: Cool thanks!

**Pull request merged**